dmglib/types.go

```diff
@@ -0,0 +1,16 @@
+package dmglib
+
+import (
+	"io"
+)
+
+type ReaderSeeker interface {
```

Comment:
It'd be good to document these types. IIRC these are used to "ease" testing.

That said, I am wondering what the `ReaderSeeker` is. The stdlib already provides `ReadSeeker`: https://pkg.go.dev/io#ReadSeeker

Reply:
I'll document them. It looks like there's no compound types in the stdlib that include `ReaderAt`?